dashboards/PostgreSQL_Overview.json

```diff
@@ -116,9 +116,9 @@
     ],
     "panels": [
         {
-            "content": "<h1><i><font color=#5991A7><b><center>Data for </font><font color=#e68a00>$host</font> <font color=#5991A7> with</font> </font><font color=#e68a00>$interval</font> <font color=#5991A7>resolution</center></b></font></i></h1><br>\n<h2><i><font color=#e68a00><center>$version</center></font></i></h2>",
+            "content": "<h1><i><font color=#5991A7><b><center>Data for </font><font color=#e68a00>$host</font> <font color=#5991A7> with</font> </font><font color=#e68a00>$interval</font> <font color=#5991A7>resolution</center></b></font></i></h1><br>",
             "gridPos": {
-                "h": 4,
+                "h": 3,
```

Comment:
can this be 2? as we used before, to save some space?

Reply:
Done